doc/architecture.rst

```diff
@@ -418,15 +418,15 @@ the greater cluster provides several benefits:
    This assures that Ceph Monitors are lightweight processes.  See `Monitoring
    OSDs`_ and `Heartbeats`_ for additional details.
 
-#. **Data Scrubbing:** As part of maintaining data consistency and cleanliness, 
-   Ceph OSD Daemons can scrub objects. That is, Ceph OSD Daemons can compare
-   their local objects metadata with its replicas stored on other OSDs. Scrubbing
-   happens on a per-Placement Group base. Scrubbing (usually performed daily)
-   catches mismatches in size and other metadata. Ceph OSD Daemons also perform deeper
-   scrubbing by comparing data in objects bit-for-bit with their checksums.
-   Deep scrubbing (usually performed weekly) finds bad sectors on a drive that
-   weren't apparent in a light scrub. See `Data Scrubbing`_ for details on
-   configuring scrubbing.
+#. **Data Scrubbing:** To maintain data consistency, Ceph OSD Daemons scrub
+   objects. Ceph OSD Daemons compare the metadata of their own local objects
```

Comment:
RADOS objects, I think if we disambiguate them once up front that suffices.

Reply:
Yeah, that's a good point. Accepted.